### PR TITLE
Verbosity parameter (if True) prints info at each iteration.

### PR DESCRIPTION
**Pull request by fdtomasi**
To achieve the same behaviour as before, the default verbosity parameter is now False in both BayesianOptimization and BO classes.

### CODE REVIEW

**Comment by javiergonzalezh** [comment]
Hi @fdtomasi, 
What do you mean when you say 'same behavior as before'? Can you cite a particular commit? Happy to merge if we are changing the default.

**Comment by fdtomasi** [comment]
I mean that the default `verbosity` parameter was previously set to `True`, even though it did not print anything.
Now, to avoid changing the default behaviour (that is, a "silent" execution, without `print`-s), I changed the default value of the verbosity parameter to `False`.

File: GPyOpt/core/bo.py [hunk 1]
```diff
@@ -149,6 +149,9 @@ def run_optimization(self, max_iter = 0, max_time = np.inf,  eps = 1e-8, context
             self.cum_time = time.time() - self.time_zero
             self.num_acquisitions += 1
 
+            if verbosity:
+                print("num acquisition: %d, time elapsed: %.2fs" % (
```
Comment (apaleyes) [inline GPyOpt/core/bo.py]: Can you use `"".format()` here? It is a preferred way of doing string formats in python these days

Comment (fdtomasi) [reply GPyOpt/core/bo.py]: Sure thing.

**Comment by fdtomasi** [comment]
In my opinion in general I find it better to have a silent execution unless explicitly requested by the user.
Of course if you prefer I can change the default back to `True`. I'll wait for a confirm.

**Comment by apaleyes** [comment]
Thinking about it a bit more, I tend to agree. And this is not a change in functionality anyway. The only worry is that this change may come across as a surprise for some users.

**Comment by fdtomasi** [comment]
I guess that there was no use for the parameter before, so it was never really used. 
If someone was setting explicitly `verbosity=True` nothing was happening, while I guess they were expecting some `print`-s happening.
If they were setting `verbosity=False`, instead, they were not expecting any `print` to happen, which is a behaviour that will still hold.
So if they were keeping the default `verbosity` and they were happy with it, I guess they would like the behaviour of `verbosity=False`.

**Comment by apaleyes** [comment]
Convincing argument! Let's merge

**Comment by fdtomasi** [comment]
Thank you.
I see now I forgot to update the docstring of `run_optimization`, where it is specified "(default, True)" of the `verbosity` parameter (line 78).

**Comment by apaleyes** [comment]
@fdtomasi can you do another PR for that please?

**Comment by fdtomasi** [comment]
Sure, done in #148 .

**Pull request merged**